tests/python/shared/fixtures/init.py

```diff
@@ -22,6 +22,19 @@
 CVAT_DB_DIR = ASSETS_DIR / "cvat_db"
 PREFIX = "test"
 
+DB_CONTAINER = "cvat_db"
+SERVER_CONTAINER = "cvat_server"
+ANNOTATION_CONTAINER = "cvat_worker_annotation"
+IMPORT_CONTAINER = "cvat_worker_import"
+EXPORT_CONTAINER = "cvat_worker_export"
+
+CODE_COVERED_CONTAINERS = [
+    SERVER_CONTAINER,
+    ANNOTATION_CONTAINER,
+    IMPORT_CONTAINER,
+    EXPORT_CONTAINER,
+]
```

Comment:
Why we don't collect coverage from cvat_worker_webhooks?

Reply:
Added